website/src/pages/download.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import Layout from '@theme/Layout';
+import Link from '@docusaurus/Link';
+
+import { Column, Container, Row } from '../components/Grid';
+import Sponsors from '../components/Sponsors';
+
+
+const LATEST_VERSION = '5.0.0-alpha15';
+
+
+const DownloadLink = ({ arch }) => {
+	let filename, label;
+	switch (arch) {
+	case 'macos':
+		filename = 'macosx.tar.gz';
+		label = 'MacOS';
+		break;
+	case 'linux':
+		filename = 'linux.tar.gz';
+		label = 'Linux';
+		break;
+	case 'src':
+		filename = 'src.zip';
+		label = 'Source Code';
+		break;
+	case 'windows':
+		filename = 'windows.zip';
+		label = 'Windows';
+		break;
+	}
+	return (
+		<div className="download-link">
+			<Link to={`https://github.com/premake/premake-core/releases/download/v${LATEST_VERSION}/premake-${LATEST_VERSION}-${filename}`}>
+				<b>{label}</b>
+			</Link>
+		</div>
+	);
+};
+
+
+const Download = () =>
+	<Layout title="Download">
+		<main className="download">
+			<Container className="intro">
+				<Row>
+					<Column>
+						<h1>Download Premake</h1>
+						<p>
+							Premake is a self-contained, single file command line executable which should build and run pretty much everywhere.
+							See <Link to="/docs/using-premake">Using Premake</Link> for usage instructions and help getting started.
+						</p>
+						<p>
+							The latest released version is <b>v{LATEST_VERSION}</b>. <Link to="https://github.com/premake/premake-core/releases">See all releases</Link>.
+						</p>
+					</Column>
+				</Row>
+			</Container>
+			<Container>
+				<Row>
+					<Column>
+						<h3>Pre-Built Binaries</h3>
+						<p>Binaries simply need to be unpacked and placed somewhere on the system search path or any other convenient location.</p>
+						<p>
+							<DownloadLink arch="windows" /><br />
+							<DownloadLink arch="linux" /><br />
+							<DownloadLink arch="macos" /><br />
+						</p>
+					</Column>
+					<Column>
+						<h3>Build It Yourself</h3>
+						<p>
+							The source code package includes project files for all supported toolsets.
+							See <Link to="https://github.com/premake/premake-core/blob/master/BUILD.txt">BUILD.txt</Link> for
+							build instructions.
+						</p>
+						<p>
+							<DownloadLink arch="src" />
+						</p>
```

Comment:
Same problem as above. I suggest using list with one item. Left padding is natural in lists. You don't have to create it artificially for every `DownloadLink`.